src/com/example/pathom.cljs

```diff
@@ -77,20 +61,27 @@
     {:current-user
      (conj {:user/id user-id} renamed-meta)}))
 
-(pc/defmutation create-random-thing [env {:keys [tmpid] :as params}]
-  ;; Fake generating a new server-side entity with
-  ;; a server-decided actual ID
-  ;; NOTE: To match with the Fulcro-sent mutation, we
-  ;; need to explicitly name it to use the same symbol
-  {::pc/sym 'com.example.mutations/create-random-thing
-   ::pc/params [:tempid]
+(pc/defmutation create-readiness-response [env {user-id :user/id response-total :response/total response-id :response/id answers :answer/id :as params}]
+  {::pc/sym 'com.example.mutations/create-readiness-response
+   ::pc/params [:user/id :response/id :response/total {:answer/id [:answer/score :question/id :user/id]}]
    ::pc/output [:tempids]}
-  (println "SERVER: Simulate creating a new thing with real DB id 123" tmpid)
-  {:tempids {tmpid 123}})
+  (go
+    (let [response (-> (<! (insert-response {:user/id user-id :response/total response-total}))
+                       (js->clj :keywordize-keys true)
+                       :data
+                       first)
+          reified-answers (-> (<! (insert-response-answers {:answers answers :response/id (:id response)}))
+                              (js->clj :keywordize-keys true)
+                              :data)
```

Comment:
once we store the response, we can store the answers. Answers need a `:response/id` on them so they are dependent.